### ISSUE TITLE
Python 3.5 Windows wheel build broken

### ISSUE THREAD

**Issue by JukkaL** [python/mypy#9501]
The Appveyor build is failing on Python 3.5.

Example build: https://ci.appveyor.com/project/mypyc/mypy-mypyc-wheels/builds/35457632

Extract from build output:
```
...
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.16.27023\bin\HostX64\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MT -IC:\projects\mypy-mypyc-wheels\mypy\mypyc\lib-rt -Ibuild -IC:\Python35-x64\include -IC:\Python35-x64\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.16.27023\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.16.27023\include" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcbuild\__native_api.c /Fobuild\temp.win-amd64-3.5\Release\build\__native_api.obj /O2 /wd4102 /wd4101 /wd4146 /GL- /wd9025
cl : Command line warning D9025 : overriding '/GL' with '/GL-'
cl : Command line warning D9014 : invalid value '9025' for '/wd'; assuming '5999'
__native_api.c
build\__native_api.c(847): error C2065: 'int32_t': undeclared identifier
build\__native_api.c(847): error C2146: syntax error: missing ';' before identifier 'cpy_r_r12'
build\__native_api.c(847): error C2065: 'cpy_r_r12': undeclared identifier
build\__native_api.c(904): error C2065: 'cpy_r_r12': undeclared identifier
build\__native_api.c(906): error C2065: 'cpy_r_r12': undeclared identifier
build\__native_api.c(1044): error C2065: 'int32_t': undeclared identifier
build\__native_api.c(1044): error C2146: syntax error: missing ';' before identifier 'cpy_r_r10'
build\__native_api.c(1044): error C2065: 'cpy_r_r10': undeclared identifier
build\__native_api.c(1047): error C2065: 'int32_t': undeclared identifier
build\__native_api.c(1047): error C2146: syntax error: missing ';' before identifier 'cpy_r_r13'
build\__native_api.c(1047): error C2065: 'cpy_r_r13': undeclared identifier
build\__native_api.c(1055): error C2065: 'int32_t': undeclared identifier
build\__native_api.c(1055): error C2146: syntax error: missing ';' before identifier 'cpy_r_r21'
build\__native_api.c(1055): error C2065: 'cpy_r_r21': undeclared identifier
build\__native_api.c(1058): error C2065: 'int32_t': undeclared identifier
...
```

It looks like 3.5 uses an older version of MSVC and it's missing some features that mypyc uses (at least `int32_t`).

Relevant stackoverflow: https://stackoverflow.com/questions/5657825/whats-the-equivalent-of-int32-t-in-visual-c

**Comment by JukkaL** [comment]
Actually the MSVC version doesn't seem to have changed between Python 3.5 and 3.6. I suspect that Python 3.5 headers don't include `stdint.h` on Windows, but later versions include it. I'm going to try explicitly including it.

**Comment by JukkaL** [comment]
Fixed by #9502.
